website/src/pages/contact/sales.tsx

```diff
@@ -24,57 +24,56 @@ export default class Trial extends React.Component<any, any> {
     public render(): JSX.Element | null {
         return (
             <Layout location={this.props.location}>
-                <section className="d-flex flex-column flex-lg-row sales">
-                    <div className="form-area">
-                        <div>
-                            <h1>Contact our team</h1>
-                            <p>
-                                Let us know which <Link to="/pricing">paid product</Link> you're interested in and for
-                                how many users. Our team is happy to answer any questions. Fill out the form and we’ll
-                                be in touch as soon as possible.
-                            </p>
-                        </div>
-                        <div className="form">
-                            <div id="hubspotTrialForm" className="d-flex justify-center" />
+                <div>
+                    <div className="jumbotron text-right dark-7">
+                        <div className="row">
+                            <div className="sales py-0 d-flex flex-grow flex-column flex-lg-row align-items-center justify-content-around">
+                                <img className="logo" src="/sourcegraph/sourcegraph-mark.png" alt="logo" />
+                                <div>
+                                    <h1 className="subtitle">Try Sourcegraph Enterprise free for 30 days</h1>
+                                    <p className="text-center">
+                                        Sourcegraph is a fast, solid, full-featured code navigation engine.
```

Comment:
This subheader seems a bit strange here. Maybe instead make this a link back to the pricing page? Or just remove it, and add a link to pricing to the footer of the section on the left side

Reply:
(and the link to the pricing page can be something like "learn more about products and packaging", not just "pricing", to illustrate that it explains what "Enterprise" actually includes)

Reply:
I'll remove it. There's several links to the pricing page here so I don't think it's necessary.